backend/src/graphql/resolver/ContributionLinkResolver.ts

```diff
@@ -39,35 +40,30 @@ export class ContributionLinkResolver {
     }: ContributionLinkArgs,
   ): Promise<ContributionLink> {
     isStartEndDateValid(validFrom, validTo)
+    // TODO: this should be enforced by the schema.
```

Comment:
Same as in contribution link resolver. `name` can be `''` and therefore evaluates to true. But this is checked later in line 47